File: Data Structures & Algorithms/anagram-groups/submission-0.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
        Map<String, List<String>> result = new HashMap<>();
        for(String s : strs) {
            int[] count = new int[26];
            for(char ch : s.toCharArray()) {
                count[(int)ch - (int)'a'] += 1;
            }
            String key = Arrays.toString(count);
            result.putIfAbsent(key, new ArrayList<>());
            result.get(key).add(s);
        }
        return new ArrayList<>(result.values());
    }

}

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        result = []
        for i in range(0, len(temperatures)):
            count = 0
            found = False
            for j in range(i, len(temperatures)):
                print(temperatures[i], temperatures[j])
                if temperatures[j] > temperatures[i]:
                    found = True
                    break
                count += 1
            
            if found:
                result.append(count)
            else:
                result.append(0)
        return result
                